state_transfer/transport/stunnel/stunnel.go

```diff
@@ -25,66 +27,38 @@ type StunnelTransport struct {
 	direct           bool
 }
 
-func (s *StunnelTransport) SetCA(b *bytes.Buffer) {
-	s.ca = b
+func NewStunnelTransport() transport.Transport {
```

Comment:
We may even want to consider calling this NewTransport() and using the package as the identifier of which type of transport?

Reply:
Fair point